### ISSUE TITLE
No clear way to determine legs and fins configuration on Falcon9

### ISSUE THREAD

**Issue by markvital** [r-spacex/SpaceX-API#174]
I could not find out weather the landing legs and grid fins on Falcon9 were installed from JSON data.
I found  `land_success` and `landing_intent` flags in rocket->first_stage->cores[0] but it's not enough to definitely find out if booster/fins were installed.
I could not use `rocket_type` and `rocket_name` information either because there are many configurations of Falcon9 v1.1.

You can easily see all the possibilities of legs/fins on this infographic, I created a few years ago:
https://www.reddit.com/r/spacex/comments/68iw7q/all_spacex_launches_and_booster_landing_attempts/

I'm using version 3 of API.

Looking forward for your help.

**Comment by jakewmeyer** [comment]
This is data that isn't tracked currently, but should be.

We can easily add the following on each launch:

```json
  "second_stage": {
    "gridfins": true,
    "legs": true
  }
```

Abstracting away the entire rocket config might be a bit overkill for such a small change

**Comment by markvital** [comment]
Yes, that would be great!
Did you mean "first_stage" instead of second?

**Comment by jakewmeyer** [comment]
Yes, first stage 👍

**Comment by jakewmeyer** [comment]
All launches should be updated at this point 👍 

Decided to move the data in with the existing core data, to allow for different configs depending on the core.

```json
"first_stage": {
      "cores": [
        {
          "core_serial": "B1048",
          "flight": 2,
          "block": 5,
          "gridfins": true,
          "legs": true,
          "reused": true,
          "land_success": true,
          "landing_intent": true,
          "landing_type": "RTLS",
          "landing_vehicle": "LZ-4"
        }
      ]
    },
```

**Comment by jesusrp98** [comment]
Would be a good idea to include that information in the cores end point as well?

**Comment by markvital** [comment]
@jakewmeyer jakewmeyer what is `core`? 
*ELI5*

**Comment by jakewmeyer** [comment]
@jesusrp98 I thought about it initially, but a core isn't guaranteed to always use the fins and legs for every flight, especially if it's reused multiple times.

**Comment by jakewmeyer** [comment]
@markvital A core is just another name for the first stage rocket booster

**Comment by jesusrp98** [comment]
@jakewmeyer That's right. Some reused Block 3 & 4 boosters weren't featuring landing legs, since SpaceX didn't want to recover them.

**Comment by markvital** [comment]
@jakewmeyer so when this new data will be live?

**Comment by jakewmeyer** [comment]
@markvital Should already be live

**Comment by markvital** [comment]
@jakewmeyer 
Thank you!
BTW, I see some nulls for future launches. Filled it as a separate issue #178
